log4j-core/src/test/java/org/apache/logging/log4j/core/layout/Rfc5424LayoutTest.java

```diff
@@ -527,4 +532,61 @@ public void testParameterizedMessage() {
             appender.stop();
         }
     }
+
+    @Test
+    void testLayoutBuilder() {
+        for (final Appender appender : root.getAppenders().values()) {
+            root.removeAppender(appender);
+        }
+
+        final AbstractStringLayout layout = new Rfc5424Layout.Rfc5424LayoutBuilder()
+                .setFacility(Facility.LOCAL0)
+                .setId("Event")
+                .setEin("36aaa92")
```

Comment:
Probably a typo, which will cause `layout` to be `null`.

Reply:
Indeed! I fixed this and rebased the branch